windirstat/Item.cpp

```diff
@@ -42,6 +42,16 @@
 #pragma comment(lib, "crypt32.lib")
 #pragma comment(lib, "bcrypt.lib")
 
+namespace {
+    // Optimization Value: Initial reserve for the thread-local pathParts vector. 
+    // Chosen as the most efficient starting point (23) to minimize initial memory use.
+    // This value leverages the vector's exponential growth factor (1.5x/2x) to 
+    // ensure capacity efficiently grows to 256 (the closest power of two), 
+    // eliminating reallocations once the maximum capacity needed for the system's 
+    // path component limit (approx. 255) is reached.
+    constexpr int DEFAULT_PATH_RESERVE = 23;
```

Comment:
Define this in CItem and set it to 256. Unlike the autogrow, The value you pass to std::vector::reserve() is not exponential.  While automatic growth function (when you add an item and there's not enough room for it) can be exponential, std::vector::reserve() is not.  When you check capacity after doing std::vector::reserve(256), it is 256.